server/bundle.go

```diff
@@ -4,15 +4,25 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"log"
 	"net/http"
 
 	"github.com/julienschmidt/httprouter"
 
+	"github.com/ndlib/bendo/items"
 	"github.com/ndlib/bendo/store"
 )
 
 // BundleListHandler handles GET requests to "/bundle/list".
 func (s *RESTServer) BundleListHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+
+	if s.TapeUsed == false {
+		w.WriteHeader(503)
+		fmt.Fprintln(w, items.ErrNoStore)
```

Comment:
I gotta disagree on this one- ErrNoStore gives me the same message from
every routehandler which calls it.

On Thu, Aug 25, 2016 at 12:24 PM, Don Brower <notifications@github.com>
wrote:

> In server/bundle.go
> <https://github.com/ndlib/bendo/pull/136#discussion_r76276811>:
>
> >  	"github.com/ndlib/bendo/store"
> >  )
> >
> >  // BundleListHandler handles GET requests to "/bundle/list".
> >  func (s *RESTServer) BundleListHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
> > +
> > +	if s.TapeUsed == false {
> > +		w.WriteHeader(503)
> > +		fmt.Fprintln(w, items.ErrNoStore)
>
> Don't import items just to use this error. Just include a literal string
> "Tape access is temporarily disabled".
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ndlib/bendo/pull/136/files/116b45889c785d5598950eba80d6730d33f98945#r76276811>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADNGTBfwo42S-pKCjSgflMe1VW4VWvkVks5qjcHDgaJpZM4JsckT>
> .
>